src/webui/components/creator/TimeElapsed.tsx

```diff
@@ -7,16 +7,14 @@ import TimeElapsedColumn from './elementary/TimeElapsedColumn';
 export type Props = {
     creator: CreatorView;
     date: Date;
-    profileHandler: () => void;
```

Comment:
Why is this handler removed? With this change we aren't able to open the creators profile from the timeline, etc. anymore. I think we need to restore this?